packages/llamaindex/package.json

```diff
@@ -63,33 +62,141 @@
         "default": "./dist/index.js"
       },
       "require": {
-        "types": "./dist/type/index.d.ts",
-        "default": "./dist/cjs/index.js"
+        "types": "./dist/type/index.d.cts",
+        "default": "./dist/cjs/index.cjs"
       }
     },
-    "./next": {
+    "./agent": {
+      "require": {
+        "types": "./agent/dist/index.d.cts",
+        "default": "./agent/dist/index.cjs"
+      },
       "import": {
-        "types": "./dist/type/next.d.ts",
-        "default": "./dist/next.js"
+        "types": "./agent/dist/index.d.ts",
+        "default": "./agent/dist/index.js"
       },
+      "default": "./agent/dist/index.js"
+    },
+    "./cloud": {
       "require": {
-        "types": "./dist/type/next.d.ts",
-        "default": "./dist/cjs/next.js"
-      }
+        "types": "./cloud/dist/index.d.cts",
+        "default": "./cloud/dist/index.cjs"
+      },
+      "import": {
+        "types": "./cloud/dist/index.d.ts",
+        "default": "./cloud/dist/index.js"
+      },
+      "default": "./cloud/dist/index.js"
     },
-    "./internal/*": {
-      "import": "./dist/not-allow.js",
-      "require": "./dist/cjs/not-allow.js"
+    "./extractors": {
+      "require": {
+        "types": "./extractors/dist/index.d.cts",
+        "default": "./extractors/dist/index.cjs"
+      },
+      "import": {
+        "types": "./extractors/dist/index.d.ts",
+        "default": "./extractors/dist/index.js"
+      },
+      "default": "./extractors/dist/index.js"
     },
-    "./*": {
```

Comment:
@himself65 seems like bunchee does not support this wildcard exports (/*)

Adding all current entry points is not maintainable (also slows down the build process a lot)

Our solution is too only export the current key entry points as in this PR, but this is a breaking change.

Do you have any better idea to handle this issue?

Reply:
I think we have to specify all the subpath, for now...

Reply:
Thanks, I think then we just export the first level (as we do right now in this PR) and we accept that some code might break 
@thucpn please 
1. double check the docs that we're not suggesting to use an invalid export some where
2. add a minor changeset to be sure